services/wfm/src/main/java/org/openkilda/wfm/share/logger/FlowOperationsDashboardLogger.java

```diff
@@ -202,15 +207,18 @@ public void onFlowUpdate(Flow flow) {
     /**
      * Log a flow-update event.
      */
-    public void onFlowUpdate(String flowId, SwitchId srcSwitch, int srcPort, int srcVlan,
-                             SwitchId destSwitch, int destPort, int destVlan, String diverseFlowId, long bandwidth) {
+    public void onFlowUpdate(String flowId, RequestedFlow flow) {
```

Comment:
What is the reason to bring H&S specific class (RequestedFlow) into shared FlowOperationsDashboardLogger? After project restructuring, the logger can be a part of a shared module. But this dependency prevents this.